db/seeds.rb

```diff
@@ -2,56 +2,48 @@
 # The JournalData can then be loaded with the rake db:seed (or created alongside the db with db:setup).
 #
 require 'journal_data'
-# Journal attributes:
 
-# t.string :title
-# t.string :url
-# t.boolean :subscribed
-# t.date :date
-# t.integer :volume
-# t.integer :issue_number
+agent = Mechanize.new
```

Comment:
I wouldn't recommend adding live data in via seeds. The seeds are more useful as a way to populate the DB with fake data, just to get something up and visible in the app. This isn't something you must change immediately, but think about the distinction in the future.